__tests__/00.using-open.ava.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * This test demonstrates basic behavior of near-runner, making simple
+ * function calls and view calls to the contract from
+ * https://github.com/near-examples/rust-status-message
+ *
+ * Note that the same tests will be run on both a local sandbox environment and
+ * on testnet by using the `test:sandbox` and `test:testnet` scripts in
+ * package.json.
+ */
+import {ava as test, Runner} from 'near-runner-ava';
+
+test('using runner', async t => {
+  await Runner.open({rootAccount: 'meta'}, async ({root}) => {
+    t.is(root.accountId, 'meta');
```

Comment:
I think this needs more explanation. What is the `meta` account? Why is it called that? Why am I allowed to make calls as this account when using `near-runner`? 

I also don't think we should make this test number `00`. I think most people will be coming to near-runner / near-workspaces for testing purposes, at least at the beginning, so we should lead with that.

For this demonstration of `Runner.open`, I think we should show doing something a little more interesting, to try to help people understand what sort of cool scripting they might be able to do. Maybe we could create an account on testnet, or make calls to `wrap.testnet`, and add a comment about how we could hypothetically now exchange it for a different token on testnet's Ref Finance.